README.md

```diff
@@ -47,6 +47,20 @@ If it doesn't work for some reason, you may have to globally install gulp throug
 npm install -g gulp
 ```
 
+### Local Development
+* PostgreSQL
+You'll need a postgres instance to use as a development DB.
+You can use an existing database, like the instance used for the dev branch, use a database on another server, or spin up a container using docker or podman. 
+To get setup using docker, run
+```bash
+docker run --name packet-postgres -e POSTGRES_PASSWORD=mysecretpassword -d -p 5432:5432 postgres
```

Comment:
Any reason to not just toss this into a docker-compose file, [like I did for self-service](https://github.com/ComputerScienceHouse/self-service/blob/main/docker-compose.yaml)?

Reply:
I use podman, so docker-compose is not something I default to yet.